seqr/views/apis/users_api_tests.py

```diff
@@ -390,12 +393,12 @@ def test_set_password(self):
         self.mock_list_workspaces.assert_not_called()
         self.mock_get_ws_acl.assert_not_called()
 
-    def test_forgot_password(self, *args):
-        super(MixUsersAPITest, self).test_forgot_password(*args)
+    def test_forgot_password(self, *args, **kwargs):
+        super(MixUsersAPITest, self).test_forgot_password(*args, **kwargs)
```

Comment:
yes I updated codacy to use the python 3 pylint instead of python 2 pylint, and the arguments-differ error now goes away if you use `(*args, **kwargs)` notation